syntax = "proto3";

package CustomerInfoSvc;

message Customer {

    string firstName = 1;
    string lastName = 2;
    string emailId = 3;
}